pkg/ddc/juicefs/transform_fuse.go

```diff
@@ -226,22 +226,25 @@ func (j *JuiceFSEngine) genValue(mount datav1alpha1.Mount, tiredStoreLevel *data
 
 	var storagePath = DefaultCacheDir
 	var volumeType = common.VolumeTypeHostPath
+	var volumeSource datav1alpha1.VolumeSource
 	if tiredStoreLevel != nil {
 		// juicefs cache-dir use colon (:) to separate multiple paths
 		// community doc: https://juicefs.com/docs/community/command_reference/#juicefs-mount
 		// enterprise doc: https://juicefs.com/docs/cloud/commands_reference#mount
 		// /mnt/disk1/bigboot or /mnt/disk1/bigboot:/mnt/disk2/bigboot
 		storagePath = tiredStoreLevel.Path
 		volumeType = tiredStoreLevel.VolumeType
+		volumeSource = tiredStoreLevel.VolumeSource
 	}
 	originPath := strings.Split(storagePath, ":")
 
 	// transform cacheDir
 	value.CacheDirs = make(map[string]cache)
```

Comment:
How about adding `useTieredStore` flag in FUSE spec to provide a way to enable/disable configuring cacheDirs in FUSE pod?

Reply:
next PR